src/components/GoodsList.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import './GoodsList.scss';
+import PropTypes from 'prop-types';
+
+export class GoodsList extends React.Component {
+  state = {
+    goods: [...this.props.goods],
```

Comment:
remember about this

[https://stackoverflow.com/a/50403930](https://stackoverflow.com/a/50403930)

[https://reactjs.org/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state](https://reactjs.org/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state)

[https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html)

Reply:
thank for information